### ISSUE TITLE
Problem on the converge of collision value

### ISSUE THREAD

**Issue by ftyghome** [peasant98/Dronet-Pytorch#2]
Hello, thanks for your great work on the PyTorch implementation of Dronet :D 
In the training progress, I actually come with some problems and wonder if it is my fault.
I downloaded the Udacity steering dataset and the collision dataset collected on bikes, by running the move_data.py and do some small tweaks to the structure of folders, I made the dronet_torch_train work. However, the training result is not ideal. The loss plot shows that the steering value converged at the end, while the training result of the collision data looks not so good.
![image](https://user-images.githubusercontent.com/22185413/118973779-a0e3f080-b9a4-11eb-8618-fba00aa82b3a.png)
I have been trying to resolve the issue for days, but I still not find what I did wrong. Could you please offer me some advice on it?
Thanks:)

The structure of the "data" folder is as follows:
`.
`-- collision
    `-- collision_dataset
        |-- testing
        |   |-- DSCN2571
        |   |   `-- images
        |   |-- GOPR0200
        |   |   `-- images
        |   |-- GOPR0255
        |   |   `-- images
        |   |-- GOPR0265
        |   |   `-- images
        |   |-- GOPR0366
        |   |   `-- images
        |   |-- GOPR0369
        |   |   `-- images
        |   |-- GOPR0382
        |   |   `-- images
        |   |-- GOPR0386
        |   |   `-- images
        |   `-- UDACITY_TESTING
        |       `-- images
        |-- training
        |   |-- DSCN2561
        |   |   `-- images
        |   |-- DSCN2562
        |   |   `-- images
        |   |-- DSCN2563
        |   |   `-- images
        |   |-- DSCN2564
        |   |   `-- images
        |   |-- DSCN2565
        |   |   `-- images
        |   |-- DSCN2566
        |   |   `-- images
        |   |-- DSCN2567
        |   |   `-- images
        |   |-- DSCN2568
        |   |   `-- images
        |   |-- DSCN2569
        |   |   `-- images
        |   |-- DSCN2573
        |   |   `-- images
        |   |-- DSCN2574
        |   |   `-- images
        |   |-- DSCN2575
        |   |   `-- images
        |   |-- DSCN2677
        |   |   `-- images
        |   |-- DSCN2678
        |   |   `-- images
        |   |-- DSCN2679
        |   |   `-- images
        |   |-- DSCN2680
        |   |   `-- images
        |   |-- DSCN2681
        |   |   `-- images
        |   |-- DSCN2683
        |   |   `-- images
        |   |-- DSCN2684
        |   |   `-- images
        |   |-- DSCN2685
        |   |   `-- images
        |   |-- DSCN2688
        |   |   `-- images
        |   |-- DSCN2689
        |   |   `-- images
        |   |-- DSCN2692
        |   |   `-- images
        |   |-- DSCN2693
        |   |   `-- images
        |   |-- DSCN2694
        |   |   `-- images
        |   |-- DSCN2697
        |   |   `-- images
        |   |-- GOPR0201
        |   |   `-- images
        |   |-- GOPR0203
        |   |   `-- images
        |   |-- GOPR0204
        |   |   `-- images
        |   |-- GOPR0212
        |   |   `-- images
        |   |-- GOPR0213
        |   |   `-- images
        |   |-- GOPR0214
        |   |   `-- images
        |   |-- GOPR0216
        |   |   `-- images
        |   |-- GOPR0217
        |   |   `-- images
        |   |-- GOPR0219
        |   |   `-- images
        |   |-- GOPR0220
        |   |   `-- images
        |   |-- GOPR0221
        |   |   `-- images
        |   |-- GOPR0223
        |   |   `-- images
        |   |-- GOPR0224
        |   |   `-- images
        |   |-- GOPR0225
        |   |   `-- images
        |   |-- GOPR0226
        |   |   `-- images
        |   |-- GOPR0228
        |   |   `-- images
        |   |-- GOPR0229
        |   |   `-- images
        |   |-- GOPR0230
        |   |   `-- images
        |   |-- GOPR0231
        |   |   `-- images
        |   |-- GOPR0232
        |   |   `-- images
        |   |-- GOPR0233
        |   |   `-- images
        |   |-- GOPR0234
        |   |   `-- images
        |   |-- GOPR0235
        |   |   `-- images
        |   |-- GOPR0236
        |   |   `-- images
        |   |-- GOPR0237
        |   |   `-- images
        |   |-- GOPR0238
        |   |   `-- images
        |   |-- GOPR0239
        |   |   `-- images
        |   |-- GOPR0240
        |   |   `-- images
        |   |-- GOPR0241
        |   |   `-- images
        |   |-- GOPR0242
        |   |   `-- images
        |   |-- GOPR0243
        |   |   `-- images
        |   |-- GOPR0244
        |   |   `-- images
        |   |-- GOPR0245
        |   |   `-- images
        |   |-- GOPR0246
        |   |   `-- images
        |   |-- GOPR0247
        |   |   `-- images
        |   |-- GOPR0249
        |   |   `-- images
        |   |-- GOPR0250
        |   |   `-- images
        |   |-- GOPR0251
        |   |   `-- images
        |   |-- GOPR0252
        |   |   `-- images
        |   |-- GOPR0253
        |   |   `-- images
        |   |-- GOPR0254
        |   |   `-- images
        |   |-- GOPR0256
        |   |   `-- images
        |   |-- GOPR0259
        |   |   `-- images
        |   |-- GOPR0260
        |   |   `-- images
        |   |-- GOPR0261
        |   |   `-- images
        |   |-- GOPR0262
        |   |   `-- images
        |   |-- GOPR0263
        |   |   `-- images
        |   |-- GOPR0264
        |   |   `-- images
        |   |-- GOPR0266
        |   |   `-- images
        |   |-- GOPR0267
        |   |   `-- images
        |   |-- GOPR0268
        |   |   `-- images
        |   |-- GOPR0269
        |   |   `-- images
        |   |-- GOPR0270
        |   |   `-- images
        |   |-- GOPR0271
        |   |   `-- images
        |   |-- GOPR0272
        |   |   `-- images
        |   |-- GOPR0273
        |   |   `-- images
        |   |-- GOPR0274
        |   |   `-- images
        |   |-- GOPR0275
        |   |   `-- images
        |   |-- GOPR0276
        |   |   `-- images
        |   |-- GOPR0277
        |   |   `-- images
        |   |-- GOPR0278
        |   |   `-- images
        |   |-- GOPR0297
        |   |   `-- images
        |   |-- GOPR0298
        |   |   `-- images
        |   |-- GOPR0299
        |   |   `-- images
        |   |-- GOPR0300
        |   |   `-- images
        |   |-- GOPR0301
        |   |   `-- images
        |   |-- GOPR0302
        |   |   `-- images
        |   |-- GOPR0303
        |   |   `-- images
        |   |-- GOPR0347
        |   |   `-- images
        |   |-- GOPR0349
        |   |   `-- images
        |   |-- GOPR0350
        |   |   `-- images
        |   |-- GOPR0351
        |   |   `-- images
        |   |-- GOPR0352
        |   |   `-- images
        |   |-- GOPR0353
        |   |   `-- images
        |   |-- GOPR0354
        |   |   `-- images
        |   |-- GOPR0356
        |   |   `-- images
        |   |-- GOPR0357
        |   |   `-- images
        |   |-- GOPR0358
        |   |   `-- images
        |   |-- GOPR0359
        |   |   `-- images
        |   |-- GOPR0360
        |   |   `-- images
        |   |-- GOPR0361
        |   |   `-- images
        |   |-- GOPR0362
        |   |   `-- images
        |   |-- GOPR0363
        |   |   `-- images
        |   |-- GOPR0364
        |   |   `-- images
        |   |-- GOPR0365
        |   |   `-- images
        |   |-- GOPR0367
        |   |   `-- images
        |   |-- GOPR0368
        |   |   `-- images
        |   |-- GOPR0370
        |   |   `-- images
        |   |-- GOPR0371
        |   |   `-- images
        |   |-- GOPR0373
        |   |   `-- images
        |   |-- GOPR0374
        |   |   `-- images
        |   |-- GOPR0375
        |   |   `-- images
        |   |-- GOPR0376
        |   |   `-- images
        |   |-- GOPR0377
        |   |   `-- images
        |   |-- GOPR0378
        |   |   `-- images
        |   |-- GOPR0379
        |   |   `-- images
        |   |-- GOPR0381
        |   |   `-- images
        |   |-- GOPR0383
        |   |   `-- images
        |   |-- GOPR0384
        |   |   `-- images
        |   |-- GOPR0385
        |   |   `-- images
        |   |-- GOPR0387
        |   |   `-- images
        |   `-- UDACITY_TRAINING
        |       `-- images
        `-- validation
            |-- DSCN2682
            |   `-- images
            |-- GOPR0227
            |   `-- images
            `-- UDACITY_VALIDATION
                `-- images

285 directories
`

**Comment by ftyghome** [comment]
sorry for the duplicated issues, it is due to network problem:(
